services/graph/pkg/unifiedrole/unifiedrole.go

```diff
@@ -451,19 +451,19 @@ func displayName(role *conversions.Role) *string {
 	var displayName string
 	switch role.Name {
 	case conversions.RoleViewer:
-		displayName = "Viewer"
+		displayName = "Can view"
```

Comment:
not a fan to have the same displayName for the default viewer and space viewer,
is there a way to differentiate the name somehow?

or is it the context that decided the abilities and the naming is basically the same?

Reply:
It actually was a deliberate choice for both names to be the same. The context where these roles are being used (= in the client) already gives information about whether the resource is a space or not. So from that point of view it would be redundant to have this information in the role itself.

I don't know if it makes sense to merge the default and the space viewer role into one role? Not now, but in the long run. I guess whoever tackles https://github.com/owncloud/ocis/issues/8331 will re-think the existing roles and merge them, if there is no reason for them to be separated.

Reply:
thanks for explaining, right now consolidating wont work... but yea maybe in the future